CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 ## version 1.16.0 (2020-08-25)
 
-- Adds `Parsimmon.flags(regexp)` export
-- Fixes `Parsimmon.flags(regexp)` to work in older browsers (Edge versions 15-18)
+- Fixes `flags(regexp)` to work in older browsers (Edge versions 15-18), which fixes crashes on startup in those old clients
```

Comment:
I think you could shorten this to `Fixes a crash in MS Edge 15-18 when using Parsimmon.regexp`

Also feel free to add your github handle to the changelog here as credit if you'd like 😄

Reply:
sure, solved, and thanks!